xxh: add page (#5708)
tldr-pages/tldr: pages/common/xxh.md

# xxh

> Bring your shell with all of your customizations through SSH sessions.
> Note: xxh does not install anything into system directories on the target machine; removing `~/.xxh` will clear all traces of xxh on the target machine.
> More information: <https://github.com/xxh/xxh>.

- Connect to a host and run the current shell:

`xxh "{{host}}"`

- Install the current shell into the target machine without prompting:

`xxh "{{host}}" ++install`

- Run the specified shell on the target machine:

`xxh "{{host}}" ++shell {{xonsh|zsh|fish|bash|osquery}}`

> - Use the specified directory as the location of the xxh configuration files on the target machine:

Reviewer: I feel this description is longer than it need to be. :thinking: 
```suggestion
- Use a specific xxh configuration directory on the target machine:
```
